test/lib/fabfile.py

```diff
@@ -16,15 +16,10 @@ def make_gobgp_ctn(ctx, tag='gobgp',
 
     c = CmdBuffer()
     c << 'FROM {0}'.format(from_image)
-    c << 'COPY gobgp/gobgpd /go/bin/gobgpd'
-    c << 'COPY gobgp/gobgp /go/bin/gobgp'
+    c << 'COPY gobgpd /go/bin/gobgpd'
+    c << 'COPY gobgp /go/bin/gobgp'
 
```

Comment:
Why the above change is necessary?

Reply:
The current dockerfile puts quite unnecessary restrictions on the folder structure:
1. Docker uses the whole `/clone/path/for/gobgp/..` for it's context. If this directory contains a lot of files (say Linux kernel git repo), docker build may take extremely long time
1. Gobgp sources *must* be in `gobgp` folder, something like `gobgp-custom` won't work due to hardcoded paths in the dockerfile

In this PR I try to fix both of there issues

Reply:
The current code avoids to create Dockerfile inside gobgp directory. I think that when this code was implemented, it's a recommended way (for the performance, I can't recall). It's not the case anymore?

Reply:
In terms of docker build performance, you want to reduce docker [build context](https://docs.docker.com/build/concepts/context/) as much as possible. Putting it in the parent directory achieves the opposite

For me the following experiment shows docker reading a neighboring directory to gobgp source
```bash
mkdir ../not-a-gobgp-dir
cd ..
inotifywait -r -m  not-a-gobgp-dir &
cd -
fab -r ./test/lib make-gobgp-ctn
```